project/src/test/java/com/google/sps/EventRankerTest.java

```diff
@@ -0,0 +1,115 @@
+package com.google.sps;
+
+import com.google.cloud.Date;
+import com.google.sps.data.Event;
+import com.google.sps.data.User;
+import com.google.sps.data.VolunteeringOpportunity;
+import com.google.sps.utilities.EventRanker;
+import com.google.sps.utilities.TestUtils;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Tests the ranking algorithm for event discovery */
+@RunWith(JUnit4.class)
+public final class EventRankerTest {
+  private static final String CONSERVATION = "conservation";
+  private static final String FOOD = "food";
+  private static final String MUSIC = "music";
+  private static final String SEWING = "sewing";
+  private static final Set<String> INTERESTS_CONSERVATION_FOOD =
+      new HashSet<>(Arrays.asList(CONSERVATION, FOOD));
+  private static final Set<String> SKILLS_MUSIC = new HashSet<>(Arrays.asList(MUSIC));
+  private static Event EVENT_CONSERVATION_FOOD_MUSIC;
+  private static Event EVENT_FOOD_MUSIC;
+  private static Event EVENT_CONSERVATION_MUSIC;
+  private static Event EVENT_FOOD;
+  private static Event EVENT_SEWING;
+  private static User USER_CONSERVATION_FOOD_MUSIC;
+  private static VolunteeringOpportunity OPPORTUNITY_MUSIC;
+
+  @BeforeClass
+  public static void setUp() throws Exception {
+    int currentYear = new java.util.Date().getYear();
+    EVENT_CONSERVATION_FOOD_MUSIC =
+        TestUtils.newEvent().toBuilder()
+            .setLabels(new HashSet<>(Arrays.asList(CONSERVATION, FOOD, MUSIC)))
+            .build();
+    EVENT_FOOD_MUSIC =
+        TestUtils.newEvent().toBuilder()
+            .setLabels(new HashSet<>(Arrays.asList(FOOD, MUSIC)))
+            .setDate(Date.fromYearMonthDay(currentYear + 1, 1, 1))
```

Comment:
This information of the year is purposely being set to create a specific set of result, so should be moved out of setup?
ex. currentYear + 1 etc

Reply:
Done